src/server.js

```diff
@@ -29,7 +31,10 @@ module.exports = function() {
 	var app = express()
 		.use(allRequests)
 		.use(index)
-		.use(express.static("client"));
+		.use(express.static("client"))
+		.engine("html", expressHandlebars({extname: ".html", helpers: require("../client/js/libs/handlebars")}))
+		.set("view engine", "html")
+		.set("views", path.join(__dirname, "..", "client"));
```

Comment:
Sorry, I highlighted the wrong line, but I'm glad you detailed this as well. I agree though I need to read up a bit on it to make sure I'm following.

What I meant to highlight was the fact that we require a client subfolder in a server file... I don't know, feels wonky. I won't hold on that obviously, just sharing thoughts.